examples/audio/audio_raw_stream.py

```diff
@@ -0,0 +1,104 @@
+from cffi import FFI
+from pyray import *
+from math import sin, pi, floor
+
+MAX_SAMPLES = 512
+MAX_SAMPLES_PER_UPDATE = 4096
+
+
+frequency = 440.0
+audio_frequency = 440.0
+old_frequency = 1.0
+sine_idx = 0.0
+
+# Need to wrap audio callback fucntion in ffi callback
+AUDIO_CALLBACK = ffi.callback("void(void *, unsigned int)")
```

Comment:
i think this would better be done with a decorator like this? https://cffi.readthedocs.io/en/stable/using.html#extern-python-new-style-callbacks